catalog-info.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: kafka-connect-storage-cloud
+  description: Kafka connector for loading data to S3.
+  annotations:
+    github.com/project-slug: 'peak-platform/kafka-connect-storage-cloud'
+    backstage.io/techdocs-ref: url:https://github.com/peak-platform/kafka-connect-storage-cloud
+  tags:
+    - java
+    - fork
```

Comment:
Should we also make it private as we are doing with kafka-connect-jdbc?

Reply:
This has no security vulnerability, but we can do it.

Reply:
Thanks. We can update the status of this repo in Sheet and migrate it to new repo as kafka-connect-jdbc-connectors is also dependent on this.